examples/Tutorials/Boilerplate/JustRide/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  JustRide
+//
+//  Created by Charlyn Gonda on 8/24/16.
+//  Copyright © 2016 Charlyn Gonda. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
might want to add an `import UberRides` since this is a boilerplate for an app that uses it